diff --git a/node_modules/pinia/dist/pinia.mjs b/node_modules/pinia/dist/pinia.mjs
index 5b08908..66292a3 100644
--- a/node_modules/pinia/dist/pinia.mjs
+++ b/node_modules/pinia/dist/pinia.mjs
@@ -527,7 +527,7 @@ const getStoreType = (id) => '🍍 ' + id;
  * @param app - Vue application
  * @param pinia - pinia instance
  */
-function registerPiniaDevtools(app, pinia) {
+export function registerPiniaDevtools(app, pinia) {
     setupDevtoolsPlugin({
         id: 'dev.esm.pinia',
         label: 'Pinia 🍍',
@@ -953,7 +953,7 @@ function patchActionForGrouping(store, actionNames, wrapWithProxy) {
 /**
  * pinia.use(devtoolsPlugin)
  */
-function devtoolsPlugin({ app, store, options }) {
+export function devtoolsPlugin({ app, store, options }) {
     // HMR module
     if (store.$id.startsWith('__hot:')) {
         return;
