diff --git a/node_modules/pinia/dist/pinia.mjs b/node_modules/pinia/dist/pinia.mjs
index 4a25b54..ca2fac9 100644
--- a/node_modules/pinia/dist/pinia.mjs
+++ b/node_modules/pinia/dist/pinia.mjs
@@ -536,7 +536,7 @@ const getStoreType = (id) => '🍍 ' + id;
  * @param app - Vue application
  * @param pinia - pinia instance
  */
-function registerPiniaDevtools(app, pinia) {
+export function registerPiniaDevtools(app, pinia) {
     setupDevtoolsPlugin({
         id: 'dev.esm.pinia',
         label: 'Pinia 🍍',
@@ -962,7 +962,7 @@ function patchActionForGrouping(store, actionNames, wrapWithProxy) {
 /**
  * pinia.use(devtoolsPlugin)
  */
-function devtoolsPlugin({ app, store, options }) {
+export function devtoolsPlugin({ app, store, options }) {
     // HMR module
     if (store.$id.startsWith('__hot:')) {
         return;
